docker/docker-compose.yml

```diff
@@ -115,12 +113,10 @@ services:
       REGION: stub
       GLOBAL_S3_BUCKET: stub
     volumes:
-      - type: volume
-        source: volume_storage
-        target: /var/lib/storage
+      - ./volumes/storage:/var/lib/storage
 
-  meta:
-    container_name: supabase-meta
+  pg-meta:
+    container_name: supabase-pg-meta
```

Comment:
Kong uses `pg-meta` so reverting this too, alternatively we can change it on `kong.yml`